fedireads/goodreads_import.py

```diff
@@ -1,126 +1,61 @@
 ''' handle reading a csv from goodreads '''
-import re
 import csv
-import itertools
-import dateutil.parser
+from requests import HTTPError
 
-from fedireads import books_manager
-from fedireads.models import Edition, ReadThrough
+from fedireads import outgoing
+from fedireads.tasks import app
+from fedireads.models import ImportJob, ImportItem
+from fedireads.status import create_notification
 
-
-# Mapping goodreads -> fedireads shelf titles.
-GOODREADS_SHELVES = {
-    'read': 'read',
-    'currently-reading': 'reading',
-    'to-read': 'to-read',
-}
 # TODO: remove or notify about this in the UI
 MAX_ENTRIES = 20
```

Comment:
can we remove this limit safely now?

Reply:
I'm not sure. I'm kind of wary of having it unlimited yet. I don't like the idea that someone can upload a million line file and make us spam openlibrary with requests. I might be worrying about nothing as I'm sure there are lots of other ways to cause problems.

The real fix is to rate limit outgoing requests.

I was going to look at this as part of the error handling. How about upping this to 500 for now?

Reply:
oh good point! I'll ask Mek about rate limit/politeness settings on Monday

Reply:
okay, I talked to Mek! he says it should be okay to make lots of requests synchronously, but things can get hairy with concurrent requests. He suggested using the data dump rather than querying the API -- I'm not sure how large it is, but it's worth considering I think (although outside the scope of this PR).

Reply:
Interesting. I think it might not be reasonable to expect every instance to have a full copy of all the book data, but it's good to have that option.

I think I'd still suggest upping the limit to 500 or a 1,000 so we test it on some real data and make sure that nothing breaks. (Mine export is 267 entries, I think.)

Reply:
👍 I support trying it with that limit, but let's wait until we've tested it before committing it

Reply:
I will do. I was trying it locally but got blocked by the weird id bug.